unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/RPCSocialApiBridge.cs

```diff
@@ -138,6 +191,14 @@ private async UniTask<List<string>> GetAllFriends(CancellationToken cancellation
         {
             List<string> result = new ();
 
+            if (socialClient == null) { await WaitForSocialClient(cancellationToken); }
+
+            if (socialClient == null)
+            {
+                Debug.LogError("Getting all friends failed due to lack of client");
+                return new List<string>();
```

Comment:
i would rather prefer to throw an exception than returning an invalid value so the inner layers can react accordingly

Reply:
This is OOS, will review in the future